app/assets/javascripts/components/auto_click_link.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
+
+document.addEventListener("DOMContentLoaded", function() {
+  var url = document.querySelector("[data-auto-follow-link=true]");
```

Comment:
Caveat for all future JS review that might come in this PR: I still know very little about JS.

Would `element` be a better variable name than `url`?

Reply:
No probs! `element` seems too generic maybe `linkElement` as a compromise?

Reply:
:+1: